src/wechaty/user/contact_self.py

```diff
@@ -0,0 +1,78 @@
+from wechaty import FileBox, get_logger
+from .contact import Contact
+from typing import Optional
+import asyncio
+
+log = get_logger('contact_self')
```

Comment:
Just like any other `log`, the name of log should be capital. So the name of this log should be `ContactSelf`.